Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 ruby '2.2.2'
 
+gem 'scss_lint', require: false # require flag is necessary https://github.com/brigade/scss-lint#installation
+gem "csvlint"
+gem "puppet-lint"
+gem "reek"
 gem "rubocop"
+gem "ruby"
```

Comment:
They are, except the  `scss_lint` was added to the template file because of `require: false` part :sweat_smile: